recipe/build-mpi.sh

```diff
@@ -34,6 +34,11 @@ if [[ "$target_platform" == osx-* ]]; then
   fcdir=$($FC -print-search-dirs | awk '/install: /{print $2}')
   ccdir=$($CC -print-search-dirs | awk '/libraries: =/{print substr($2,2)}')
   ln -s $fcdir/include/ISO_Fortran_binding.h $ccdir/include
+  echo "=== DEBUG ==="
+  echo "CCDIR: ${ccdir}"
+  echo "Content of ${ccdir}"
+  echo "$(ls ${ccdir})"
```

Comment:
Maybe you meant the following?

```suggestion
  echo "Content of ${ccdir}/include"
  echo "$(ls ${ccdir}/include)"
```

Reply:
Hehe, yes, I just looked at results now, and yes, I meant exactly that. :smile: